packages/blade/src/components/Chip/_decisions/decisions.md

```diff
@@ -0,0 +1,127 @@
+
+## `ChipGroup` API
+
+| Prop              | Type                                                                     | Default     | Description                                                                                                                    | Required |
+| ----------------- | ------------------------------------------------------------------------ | ----------- | ------------------------------------------------------------------------------------------------------------------------------ | -------- |
+| **children**      | `React.ReactNode`                                                        | `undefined` | Accepts multiple `Chip` components                                                                                             | ✅       |
+| **defaultValue**  | `string \|string[]`                                                      | `undefined` | Sets the initial value of the Chip group                                                                                       | ❌       |
+| **isDisabled**    | `boolean`                                                                | `false`     | Disables or enables ChipGroup, it will propagate down to all the Chips                                                         | ❌       |
+| **name**          | `string`                                                                 | `undefined` | The name of the chip group, [useful in form submissions](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#name) | ❌       |
+| **onChange**      | `({ name, value }: { name: string; value: string \| string[] }) => void` | `undefined` | Calls a function on any state change within the ChipGroup                                                                      | ❌       |
+| **selectionType** | `'single' \| 'multiple'`                                                 | `'single'`  | Defines the selection behavior within the ChipGroup component                                                                  | ❌       |
+| **size**          | `'xsmall' \| 'small' \| 'medium' \| 'large'`                             | `'small'`   | Specifies the size of the rendered Chips withing the ChipGroup                                                                 | ❌       |
+| **value**         | `string \|string[]`                                                      | `undefined` | Acts as a controlled component by specifying the ChipGroup value                                                               | ❌       |
+| **variant**       | `'positive' \| 'negative' \| 'notice' \| 'information' \| 'neutral'`     | `'neutral'` | Sets the ChipGroups's visual variant, it will propagate down to all the Chips                                                  | ❌       |
```

Comment:
question: will there be a use-case where every chip will be a single variant that is not `neutral`?

Reply:
There can be. I have it as one of the open questions https://github.com/razorpay/blade/pull/1535#discussion_r1295362618

Reply:
I meant from a design standpoint, would there be use-cases (might need to check with design team)? I can think of chips being all neutral which would be the default variant. But I can't think of chips all being any other variant. If so, then I guess the control can be left to the individual Chip and not ChipGroup. 

P.S: My whole assumption could be wrong.

Reply:
We discussed this internally, and we decided that all Chips should have same intent within a ChipGroup, hence I've updated the API to remove the `intent` prop from `Chip`